clouds/tests/aws/test_aws_utils.py

```diff
@@ -0,0 +1,148 @@
+import pytest
+from typing import Dict, List
+
+from clouds.aws.aws_utils import (
+    set_and_verify_aws_credentials,
+    set_and_verify_aws_config,
+    delete_all_objects_from_s3_folder,
+    delete_bucket,
+    aws_region_names,
+    get_least_crowded_aws_vpc_region,
+)
+
+OS_ENVIRON_PATCH_STR: str = "clouds.aws.aws_utils.os.environ"
```

Comment:
either add typing to everything or remove

Reply:
done